packages/plugin-ext/package.json

```diff
@@ -14,7 +14,8 @@
     "decompress": "^4.2.0",
     "lodash.clonedeep": "^4.5.0",
     "ps-tree": "1.1.0",
-    "vscode-uri": "^1.0.1"
+    "vscode-uri": "^1.0.1",
+    "monaco-languageclient": "^0.9.0"
```

Comment:
please, rely transitively on a dependency from the monaco extension. The thing is that we don't use `monaco-languageclient`, but our own package which is not tree shaken and contains all necessary VS Code apis.

I wonder why do you need it?

Reply:
> please, rely transitively on a dependency from the monaco extension. The thing is that we don't use monaco-languageclient, but our own package which is not tree shaken and contains all necessary VS Code apis.

oh, sorry I thought it is `monaco-editor-code`

anyway `monaco-languageclient` already should be there if you depend on `@theia/monaco`

Reply:
I use ` DocumentFilter, MonacoModelIdentifier, testGlob, getLanguages` from `monaco-languageclient`, it's a problem? I can delete `"monaco-languageclient": "^0.9.0"` but is it's OK to use that types? Or I need reimplement them?

Reply:
removing a dependency will be good that we manage it one place

You can forget what I said in the first comment, it is related only to `monaco-editor-code`. I've misread it.